db/migrate/20180220180926_add_default_value_to_comment_status.rb

```diff
@@ -0,0 +1,6 @@
+class AddDefaultValueToCommentStatus < ActiveRecord::Migration
+  def change
+    change_column :comments, :status, :integer, default: 1
+    Comment.update_all ["status = ?", 1]
```

Comment:
by changing the version number in schema.rb.example will only affect new comments by setting their status =1 but this migration would have no effect on current data unless `rake db:migrate` is run on production. What do you think?